ggshield/iac/models/iac_scan_result.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Dict
+
+from marshmallow import fields, post_load
+from pygitguardian.models import ScanResult, ScanResultSchema
+
+from ggshield.iac.models.iac_policy_break import IaCPolicyBreakSchema
+
+
+class IaCScanResultSchema(ScanResultSchema):
+    policy_breaks = fields.List(fields.Nested(IaCPolicyBreakSchema), required=True)
+
+    # Override parent method to prevent using the parent deserializer
+    def make_scan_result(self, data: Dict[str, Any], **kwargs: Any) -> Dict[str, Any]:
+        return data
```

Comment:
We should raise if the method is not intended to be used.

Reply:
Ok, I will check if it's used by the preload of the parent but it shouldn't